data/human/human missions.txt

```diff
@@ -4965,3 +4965,164 @@ mission "Riley"
 			label end
 			`	You say your goodbyes, leaving to your ship and returning to the stars.`
 				decline
+
+mission "A dead man's path I"
+	name "Siblings to New China"
+	description "Bring these siblings to New China to hear their father's last will."
+	passengers 2
+	source
+		near "Merak" 3 6
+	destination
+		planet "New China"
+	to offer
+		random < 20
+	on offer
+		conversation
+			`As you walk through the spaceport, a young woman and man, who are both dressed very formally, approach you. The woman begins, "Hello, are you a spaceship captain?"`
```

Comment:
```suggestion
			`The spaceport on <origin> is lively tonight. There are porters running to and from depots with crates, crewmen buying rounds of drinks for each other around the bars before they hit the bed, and bands of Navy officers jovially greeting captains and systematically inspecting ships. The energy and self-purpose of this spaceport is on full display this one night.`
			`	The only exception to the bustle and activity are a young woman and man, both dressed very formally, who are seemingly lost and having no luck approaching anybody. You watch they drift from one person to another, each time being sent off without an answer. Finally, they come towards your direction.`
			`	Their faces show clearly that they are on their last legs of hope. It is only with reluctance that the woman says, "Excuse me, are you a spaceship captain?"`
```
This way is a bit more interesting, I reckon.

Reply:
wow, this is good